### DISCUSSION TITLE
Battery State in Charging Screen…

### DISCUSSION THREAD

**Discussion by ShadowPhyton** [OnionUI/Onion#1144] [Ideas]
I think it would be cool if the Battery percentage would be visible in the Charging screen...idk of this is managable by you guys but it would be pretty awesome to see how much the Miyoo is charged with having to boot it completely

**Comment by mdeguzis** [comment]
Even when booted, most themes or all won't show the percentage while plugged it, just the lightening bolt. A workaround is to open the battery tracker app.
